ios/PurchasesHybridCommon/PurchasesHybridCommonTests/StoreProduct+HybridAdditionsTests.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  StoreProduct+HybridAdditionsTests.swift
+//  PurchasesHybridCommonTests
+//
+//  Created by Cesar de la Vega on 12/27/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Quick
+import Nimble
+import StoreKit
+@testable import RevenueCat
+
+class StoreProductHybridAdditionsTests: QuickSpec {
+
+    private func storeProductDictionary(with locale: Locale = Locale(identifier: "en_US"),
+                                            description: String = "A product description",
+                                            productIdentifier: String = "monthly",
+                                            price: Decimal = Decimal(1.0),
+                                            title: String = "Monthly Product",
+                                            introductoryPrice: SKProductDiscount? = nil,
+                                            discounts: [SKProductDiscount] = [],
+                                            subscriptionPeriod: SKProductSubscriptionPeriod? = nil
+    ) -> [String: NSObject] {
+        let storeProduct = StoreProduct(priceLocale: locale,
+                                        localizedDescription: description,
+                                        discounts: discounts,
+                                        productIdentifier: productIdentifier,
+                                        introductoryDiscount: nil,
+                                        price: price,
+                                        productType: .autoRenewableSubscription,
+                                        localizedTitle: title,
+                                        subscriptionPeriod: subscriptionPeriod,
+                                        subscriptionGroupIdentifier: "sub_group",
+                                        introductoryPrice: introductoryPrice
+        )
+        guard let receivedDictionary = storeProduct.rc_dictionary as? [String: NSObject] else {
+            fatalError("received rc_dictionary is not in the right format")
+        }
```

Comment:
If this is crashing might as well do `as!`.
Alternatively you can make the method `throws` to make the test fail instead. Not a big deal though.